src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14SwitchExpression.java

```diff
@@ -1,12 +1,45 @@
 //non-compiled with javac: Compilable with Java14
-package com.puppycrawl.tools.checkstyle.grammar;
+package com.puppycrawl.tools.checkstyle.grammar.java14;
 
+import static java.time.Instant.*;
+
+import java.time.Instant;
+import java.util.Arrays;
 import java.util.EnumSet;
 import java.util.Set;
+import java.util.function.IntConsumer;
 import java.util.function.Supplier;
 
 public class InputJava14SwitchExpression {
 
+    static class HardToParse {
+        void foo() {
+            Instant Ascii;
+            byte[] decodabet = new byte[1];
```

Comment:
how this method code is related to switch ?

Reply:
I found this code during regression testing, line 19 and 25 caused exceptions during parsing when I was still struggling with `castExpression`  (along with the code on line 25). I put all code that gave problems in this static class to make sure that we can keep parsing it.